content/07.pathogenesis.md

```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
 
 ### Systems level approaches for understanding SARS-CoV-2 pathogenesis
 
-[Sandipan Ray volunteered to lead this section]
+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
+These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
+Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
+Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
+
+The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. 
+A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7]. 
+Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. 
+Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10/ggjr43]. 
+However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].
+
+The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 
+In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, providing some clues regarding which part of the pathogen may be recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002]. 
+Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection [@doi:10/ggk6c5].
```

Comment:
The Lancet Respiratory Med paper certainly shows hyperactivation of CD4+ and CD8+ cells (incl high expression of cytotoxic granule proteins in the CD8+s), but their overall count was reduced. The authors specifically note an expansion of the Th17 expansion among CD4+ cells and speculate that these cells may promote tissue damage. There is some evidence to support these findings in other viral infections (I can provide citations) and so this is an interesting hypothesis).

Semantic note: the overactivation of T cells is not "caused" by an increase in Th17 and cytotoxic CD8+s. Rather, these findings indicate an activated T cell phenotype. 

Another group published a study looking at T cell phenotype here: https://www.nature.com/articles/s41423-020-0401-3

My summary of the major findings --> increased levels of HLA-DR (MHC-II) and TIGIT in CD8+s. No differences in activation markers on CD4+. Interestingly, HLA-DR/TIGIT on CD8+ were observed at higher levels in the severe vs mild COVID group, providing some more evidence that CD8+ activation and may be important in pathology. To me it's a bit unclear what is happening w/ the CD4+ cells...

Reply:
I should add that the Lancet Resp Med paper only phenotyped 1 patient and therefore should be considered with some caution. The cellular and molecular immuno paper I linked followed a larger cohort and reported similar findings in the CD8+ compartment so I think that's probably a solid angle to pursue.

Reply:
Yes, the Xu et al. Lancet Respir Med. 2020 is a case report from Wuhan, China (Treatment and Research Center for Infectious Diseases). Considering a single patient report (which is the same for many other top-tier publications on 2019-nCoV at this stage) the phenomenon is stated as "could be". The findings from this paper are highly interesting. Anyway, please feel free to include needful modifications.

Reply:
Many thanks for your valuable input.

Reply:
Agreed! And my pleasure. Lots of interesting things to think about. If you think the Cellular/Molecular Immuno paper merits inclusion in this section I can add a brief comment in the main text.

Reply:
Great discussion. I would add a few other potential references of interest including:

https://www.medrxiv.org/content/10.1101/2020.02.23.20026690v1
- single-cell seq of BAL also backs a dysregulated CD8 angle
- limited sample-size, but interesting to note potential sources of local cytokine storm in lungs
- caveat = all patients had pneumonia (including mild ones) which was surprising to me

https://www.medrxiv.org/content/10.1101/2020.02.18.20024364v1
- shows clear drops in T cell numbers
- implicates PD-1 and Tim3 in exhaustion as well
- inverse correlation between some circulating cytokines and T cell numbers

https://academic.oup.com/nsr/advance-article/doi/10.1093/nsr/nwaa041/5804736
- implicates Th1 subset in severe disease
- raises role for monocyte subpopulation as well

https://www.medrxiv.org/content/10.1101/2020.02.16.20023903v1
- decreased IFNg+ CD4+ T cells in severe patients
- further evidence of proteins involved in cytokine storm in severe COVID19

https://www.nature.com/articles/s41423-020-0402-2
- evidence of increased exhaustion of cytotoxic cells (CD8+ T cells and NK cells) in severe patients
- analysis of NKG2A exhaustion 

https://academic.oup.com/cid/advance-article/doi/10.1093/cid/ciaa248/5803306
- another subset analysis looking at CD4 and CD8 naive, helper, CD4 Tregs
- higher naive, and lower memory and Tregs in severe cases

Reply:
I would also encourage conservative interpretation of the Th17 finding in Lancet Resp Med. They show CCR6+ Th17s but don't use IL-17A staining to prove they are bonafide and functional Th17s. As shown in many other references, CCR6+ populations of CD4+ T cells can have functional diversity as noted in other viral infections: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4995622/

Reply:
This is really helpful. There's a lot of good stuff here. I'd propose this. I'm going to try to suggest a revision to be more conservative about Th17. Then, perhaps, we can get @rishirajgoel to come back through and suggest edits/revision to incorporate the additional papers mentioned.

At that point, perhaps @gregszetoAI and add the breadth of the literature that he raised. How does that sound?

Reply:
The statement on the findings from Xu et al. Lancet Respir Med. 2020 is tuned down as suggested. 
I think it might be a good idea to include further details in the "Immune Response to SARS-CoV-2" subsection under in the Pathogenesis section. 
Discussions on the papers suggested by @gregszetoAI and @rishirajgoel will be most suitable for that sub-section. What do you think @cgreene and HM Rando@rando2?

Reply:
Mostly agree. Given the systems bio section, I would think at least the single-cell seq paper may fit well here.

Reply:
I agree with @rays1987. I think the pathogenesis section needs some experts in virology and immunology and it seems like you guys have a good background in that. I outlined the pathogenesis section relatively early on just to spark some idea so feel free to adjust it to your needs but I think that understanding the basics of SARS-CoV-2 entry, replication and the immune response is obviously one the heart of COVID-19 biology. My suggestion would be that the systems part of this section should be more focused on the technologies and less on the biology. Most papers take advantage of systems approaches these days so it might also be worth thinking about splitting the systems bio part into where it fits into biology to avoid redundancy.